src/main/java/org/sagebionetworks/template/repo/appconfig/AppConfigDescriptor.java

```diff
@@ -15,11 +19,17 @@ public class AppConfigDescriptor {
     @JsonCreator
     public AppConfigDescriptor(@JsonProperty(value = "appConfigName", required=true) String appConfigName,
                               @JsonProperty(value = "appConfigDescription", required=true) String appConfigDescription,
-                              @JsonProperty(value = "appConfigDefaultConfiguration", required=true) String appConfigDefaultConfiguration) {
+                              @JsonProperty(value = "appConfigDefaultConfiguration", required=true) ObjectNode appConfigDefaultConfiguration) {
 
         this.appConfigName = appConfigName;
         this.appConfigDescription = appConfigDescription;
-        this.appConfigDefaultConfiguration = appConfigDefaultConfiguration;
+
+        ObjectMapper mapper = new ObjectMapper();
+        try {
+            this.appConfigDefaultConfiguration = mapper.writeValueAsString(mapper.writeValueAsString(appConfigDefaultConfiguration));
+        } catch (JsonProcessingException e) {
+            throw new IllegalArgumentException("Failed to serialize appConfigDefaultConfiguration", e);
+        }
```

Comment:
We still need to pass a string to CloudFormation, so use Jackson to serialize the object twice, which will properly escape the JSON string.